react/src/sagas/Config.js

```diff
@@ -1,19 +1,21 @@
 
 import { put, call } from "redux-saga/effects";
-import { ajaxHeaders, checkStatus, getRequest } from "actions/common";
+import { ajaxHeaders, checkStatus, getRequest, notIE11Unauthn } from "actions/common";
 import { getConfigFail } from "actions/Config";
 
 
 export function* requestConfig () {
+    const input = document.getElementById('jsconfig_url'),
+          jsconfig_url = input ? input.value : EDUID_CONFIG_URL;
     try {
-        const input = document.getElementById('jsconfig_url'),
-              jsconfig_url = input ? input.value : EDUID_CONFIG_URL;
         console.log('Getting config from ' + jsconfig_url);
         const config = yield call(fetchConfig, jsconfig_url);
         yield put(config);
     } catch(error) {
-        console.log('Error fetching config: ' + error.toString());
-        yield put(getConfigFail(error.toString()));
+        if (notIE11Unauthn(error) === true) {
+            console.log('Error fetching config: ' + error.toString());
+            yield put(getConfigFail(error.toString()));
```

Comment:
Is there a reason that we don't just call a new function called something like putError that does what notIE11Unauthn does and instead of returning true for other browsers just does the yield put(error)?

Then the catch would again just be a simple one liner.

Reply:
That was my original code, but doing `yield put(error)` in the `notIE11Unauthn` function turns it into a generator, and it was not working.

However, while answering your comment, I've found out that there is a special syntax to call a generator from within a generator, `yield* generator()`. 

So I'll test that syntax, amend my PR and ping you when it's ready.